### PR TITLE
Instrument Controller with Markers

### PR DESCRIPTION
**Pull request by ChrBi**
Signed-off-by: Markus Thoemmes <markus.thoemmes@de.ibm.com>

This PR is for the instrumentation of the controller with the LogMarkers.

### CODE REVIEW

**Comment by ChrBi** [comment]
PPG 508 started

File: core/controller/src/whisk/core/controller/ApiUtils.scala [hunk 1]
```diff
@@ -132,12 +133,13 @@ trait ReadOps extends Directives with Messages with Logging {
      * - 500 Internal Server Error
      */
     protected def listEntities(list: Future[List[JsValue]])(implicit transid: TransactionId) = {
+        info(this, "", LoggingMarkers.CONTROLLER_LIST_START)
```
Comment (rabbah) [inline core/controller/src/whisk/core/controller/ApiUtils.scala]: these ara all redundant with the db operations - do we really need them?

File: common/scala/src/whisk/common/Logging.scala [hunk 2]
```diff
@@ -119,14 +119,44 @@ object LoggingMarkers {
...
     val CONTROLLER_ACTIVATION_END = LogMarkerToken(controller, activation, finish)
     val CONTROLLER_ACTIVATION_REJECTED = LogMarkerToken(controller, activation, "reject")
     val CONTROLLER_ACTIVATION_FAILED = LogMarkerToken(controller, activation, error)
 
-    private val waitForBlockingActivation = "waitForBlockingActivation"
-    val CONTROLLER_BLOCK_FOR_RESULT = LogMarkerToken(controller, waitForBlockingActivation, start)
-    val CONTROLLER_BLOCKING_ACTIVATION_END = LogMarkerToken(controller, waitForBlockingActivation, finish)
+    private val blockingActivation = "blockingActivation"
+    val CONTROLLER_BLOCKING_ACTIVATION_START = LogMarkerToken(controller, blockingActivation, start)
+    val CONTROLLER_BLOCKING_ACTIVATION_DONE = LogMarkerToken(controller, blockingActivation, finish)
```
Comment (rabbah) [inline common/scala/src/whisk/common/Logging.scala]: `DONE` -> `FINISH`? Isn't this the convention used on other markers?

**Comment by markusthoemmes** [comment]
merged in 92887bbf9763f3d4204d15ee53d13600d375ef08

**Pull request closed without merging**